src/pybind/mgr/dashboard/frontend/src/app/ceph/cephfs/cephfs-subvolume-snapshots-list/cephfs-subvolume-snapshots-list.component.ts

```diff
@@ -123,26 +124,11 @@ export class CephfsSubvolumeSnapshotsListComponent implements OnInit, OnChanges
 
     this.cephfsSubvolumeGroupService
       .get(this.fsName)
-      .pipe(
-        switchMap((groups) => {
-          // manually adding the group '_nogroup' to the list.
-          groups.unshift({ name: '' });
-
-          const observables = groups.map((group) =>
-            this.cephfsSubvolumeService.existsInFs(this.fsName, group.name).pipe(
-              switchMap((resp) => {
-                if (resp) {
-                  this.subvolumeGroupList.push(group.name);
-                }
-                return of(resp); // Emit the response
-              })
-            )
-          );
```

Comment:
maybe i was confusing before. we shouldn't remove this logic because this is still valid in the case of listing the snapshots. we don't want to list the groups that doesn't have a subvolume in the snapshot list page.

but in the form, we do need to list them. so the goal is to maintain two lists, one where it has all groups and the other where it has groups that has subvolumes. and preferrably we can populate these two inside a single subscription so we don't need to subscribe again. WDYT? @dnyanee1997